letta/schemas/message.py

```diff
@@ -421,8 +423,22 @@ def to_openai_dict(
 
         elif self.role == "user":
             assert all([v is not None for v in [self.text, self.role]]), vars(self)
+
+            if self.image is not None:
```

Comment:
So inside of `Message`, `.text` becomes `.content`, which basically is a 1:1 mapping to OpenAI's format: 
<img width="670" alt="image" src="https://github.com/user-attachments/assets/f0943cce-4779-4c38-bb01-4d8d80ee7190" />

https://platform.openai.com/docs/api-reference/chat/create

Reply:
agree with this and made the initial typing changes/enough to pass tests. However left off before two major changes:
1. Renaming Message.text -> Message.content which is used in a lot of places
2. Updating the ORM Message object which is typed str and requires migration. Tricky because images probably shouldn't be stored raw in the DB